src/client/features/view/components/menu/index.js

```diff
@@ -66,6 +67,12 @@ class Menu extends React.Component {
           h('div.title-container', [
             h('h4', `${this.props.name} | ${this.props.datasource}`)
           ]),
+          h('div.search-nodes', {
+            onChange : query => searchNodes(query, this.props.cy),
```

Comment:
There is a slight mismatch in naming here, query in this case should be called event or e for short, which is consistent with the react docs.  https://reactjs.org/docs/handling-events.html

You could change the function signature for searchNodes to actually take in the string value for the query.

i.e
```js
  onChange: e => searchNodes(e.target.value, this.props.cy);
```

This reduces coupling between the event and searchNodes because searchNodes doesn't have to assume it is getting a event with target.value, all it has to assume is that query is a string.

Reply:
I have changed the function signature to reduce coupling